warn/warn_control_flow_test.go

```diff
@@ -592,6 +592,34 @@ foo()
 			":7: Function \"baz\" is unused.",
 		},
 		scopeEverywhere)
+
+	checkFindings(t, "unused-variable", `
+def foo(my_iterable, arg, _some_unused_argument, _also_unused = None, *_args, **_kwargs):
+
+  a, b, _c = 1, 2, 3  # ok to not use _c
+  print(a)
+
+  _d, _e = 4, 5  # all are underscored
+  print(_d)
+
+  for f, g, _h, _ in my_iterable:  # ok to not use any underscored
+    print(f)
+
+  for _i, (_j, _k) in another_iterable:  # ok to not use any of them
```

Comment:
where is `another_iterable` defined?

Reply:
These code snippets are not being executed, so it doesn't matter, buildifier only cares that it's a syntactically valid code that can be parsed.

It could be a global variable, or a variable from some prelude file, or an error. If a variable is not defined, users are likely to detect it anyway when they execute the code, buildifier doesn't check for completely undefined variables (although it does for partially undefined, e.g. if it's defined inside `if` but not inside `else`). Even if it did check for such cases, it would be a separate warning category, out of the scope of this test case anyway.